services/kafka/sasl.go

```diff
@@ -1,9 +1,16 @@
 package kafka
 
 import (
+	"context"
 	"errors"
+	"net/url"
+	"strings"
+	"time"
 
-	sarama "github.com/IBM/sarama"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/clientcredentials"
+
+	kafka "github.com/IBM/sarama"
```

Comment:
This is very subjective. As the sarama is a Kafka client the alias makes it more clean. Also, the alias is already used in [kafka/service.go](https://github.com/influxdata/kapacitor/blob/master/services/kafka/service.go#L15)

Reply:
Well I don't like this but it might be OK if kapacitor guys agree. Let me give you my reasons:
1. You obfuscate the library used with an alias, making it harder for readers of the code to figure out what's going on.
2. Your chosen alias is the same as the package name, this will cause additional confusion.
3. You increase the PR size without _any_ necessity.

But as I said, if that's how it is done in kapacitor I don't mind.